client/card-readers/settings/index.tsx

```diff
@@ -44,21 +45,26 @@ const ReadersSettingsDescription = (): JSX.Element => (
 );
 
 const ReceiptSettings = (): JSX.Element => {
+	const [ isSaveDisabled, setDisabled ] = useState( false );
+
 	return (
 		<SettingsLayout displayBanner={ false }>
 			<SettingsSection description={ ReadersSettingsDescription }>
 				<LoadableSettingsSection numLines={ 20 }>
 					<Card className="card-readers-settings__wrapper">
 						<CardBody>
-							<BusinessDetailsSection />
+							<BusinessDetailsSection
```

Comment:
Functionally, I understand the flow and props here. But the naming is a bit confusing. At first glance, it's hard to correlate the business details section's `isDisabled` with the save button's `isSaveDisabled`.

Does the following sound better?

```suggestion
							<BusinessDetailsSection
								setSaveDisabled={ setDisabled }
							/>
```

... and create an internal state in `BusinessDetailsSection` to show/hide the error message:

```javascript
const [ hasError, setHasError ] = useState( false );

const validateBusinessURL = ( event ) => {
		if ( event.target.checkValidity() ) {
            setHasError( true );
			setSaveDisabled( false );
		} else {
            setHasError( true );
			setSaveDisabled( true );
		}
	};
```

Reply:
Marking as ready to merge, but it would be great to look into this comment @mordeth .

Reply:
Thank you, @anu-rock, your suggestion makes the readability much better. Updated.